node/core/backing/src/lib.rs

```diff
@@ -452,18 +449,12 @@ impl CandidateBackingJob {
 				&summary.candidate,
 				&self.table_context,
 			) {
-				// `HashSet::insert` returns true if the thing wasn't in there already.
-				// one of the few places the Rust-std folks did a bad job with API
-				if self.backed.insert(summary.candidate) {
-					if let Some(backed) =
-						table_attested_to_backed(attested, &self.table_context)
-					{
-						let message = ProvisionerMessage::ProvisionableData(
-							ProvisionableData::BackedCandidate(backed),
-						);
-						self.send_to_provisioner(message).await?;
-					}
-				}
+				// candidate exists and is includable, per the `attested_candidate` contract
+				// therefore, let the provisioner know about it
+				let message = ProvisionerMessage::ProvisionableData(
```

Comment:
I advocate to keep the `backed` map or we're going to send more of these than we need to. The point is that we only need to notify the provisioner _once_ and then it can request the candidates